resource/config_test.go

```diff
@@ -11,9 +11,22 @@ import (
 	"go.viam.com/rdk/components/sensor"
 	"go.viam.com/rdk/resource"
 	"go.viam.com/rdk/services/motion"
-	"go.viam.com/rdk/testutils"
 )
 
+// FakeConvertedAttributes is a helper for testing if validation works.
+type FakeConvertedAttributes struct {
```

Comment:
[drive-by] Only usage of this struct was in this file. @dgottlieb moved it here and removed `testutils/config_utils.go`.